src/rez/rezconfig.py

```diff
@@ -948,6 +948,9 @@
 #       launched without extension from windows and other systems.
 create_executable_script_mode = "single"
 
+# Default build process to use during build/release
+build_process = "local"
```

Comment:
We have `default_shell` which sets the default shell plugin to use. Should this setting be named `default_build_process` for consistency?